governance/votes/Executive Vote - October 11, 2023.md

```diff
@@ -0,0 +1,195 @@
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+- The USDP-PSM Incentives [DAO Resolution](https://gateway.pinata.cloud/ipfs/QmWg43PNNGfEyXnTv1qN8dRXFJz5ZchrmZU8qH57Ki6D62) will be approved by MKR holders.
+- RETH-A offboarding will be initiated by removing the vault from [Debt Ceiling Instant Access (`autoline`)](https://manual.makerdao.com/module-index/module-dciam) and setting the [Debt Ceiling (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to 0.
+- Reconfiguring Andromeda RWA015-A and Clydesdale RWA007-A vaults to increase their respective [Debt Ceilings (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to 3 billion DAI.
+- WBTC Vaults will have their [Debt Ceilings (`line`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) decreased and [Stability Fees](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) increased.
+- ETH Vaults will have their [Stability Fees](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) increased.
+- Qualified Aligned Voter Committee Members will receive their quarterly MKR compensation.
+- Qualified Aligned Delegates will receive their monthly MKR compensation.
+- Governance Facilitators JanSky and VoteWizard will have DAI and MKR compensation streams deployed.
+- BA Labs will receive their quarterly MKR compensation.
+- A SparkLend proxy-spell will be executed to update previously polled parameters.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### USDP-PSM Incentives
+
+*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/usdp-psm-facilitation-incentives/22331/2)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/usdp-psm-facilitation-incentives/22331) "USDP-PSM Facilitation Incentives"
+
+If this executive proposal passes, the [DAO Resolution](https://gateway.pinata.cloud/ipfs/QmWg43PNNGfEyXnTv1qN8dRXFJz5ZchrmZU8qH57Ki6D62) will be approved and in consequence, the USDP-PSM Incentives [RWA Agreement (doc)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) will be initiated with `QmWg43PNNGfEyXnTv1qN8dRXFJz5ZchrmZU8qH57Ki6D62`.
```

Comment:
I've observed that when we've updated the 'Doc' parameter in the past, it typically links to the content of the discussion thread which contains the Dao Resolution link, as evidenced by these previous instances:
https://gateway.pinata.cloud/ipfs/QmY185L4tuxFkpSQ33cPHUHSNpwy8V6TMXbXvtVraxXtb5
https://gateway.pinata.cloud/ipfs/QmWo3UVtEDKVwS5k34uLt1J6u9px3rjHYkTLK2rYQ31E3G
https://gateway.pinata.cloud/ipfs/QmYjvAZEeGCs8kMuLQz6kU8PWgsbG1i8QWd2jrwkSipcRx
https://gateway.pinata.cloud/ipfs/QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1

May I ask why the approach for this current case seems to deviate from the norm? Additionally, I noticed that the content sheet doesn't reference the "Doc" parameter but the "dao_resolution" parameter. Any insight on this?
Thank you!

Reply:
I followed Steakhouse's recommendations for the document and the copy. Would this link different from the one provided above?

Reply:
Hey @VoteWizard @MakerJanSky 

I believe this situation is different from previous instances involving a DAO resolution. Unlike, say, HV Bank or Clydesdale, the USDP-PSM is not a RWA vault. Only RWA vaults have the `doc` parameter. In this case, it is sufficient to include the IPFS hash of the DAO resolution as a constant in the code. 

In situations involving a RWA vault, the `doc` parameter must be updated like so:

https://etherscan.io/address/0xD3F96B8Ffbf21033F5A6210C6349598AAdBd1152#code#L476